scripts/bin/auto-install

```diff
@@ -1,70 +1,54 @@
 #!/bin/sh
 
-# log to SNAP_COMMON and TTY
-log() {
-    printf '%s %s\n' \
-        "$(date -Iseconds)" "$1" | tee -a "${SNAP_COMMON}/auto-install.log"
-}
-
-# Check if drive is mounted
-check_drive() {
-
-    while :; do
-        _watch_file="$SNAP_COMMON/watched"
-
-        log "Signal file is ${_watch_file}."
+mklog() {
+    # FIXME: if mklog() is called within a subshell,
+    #        then LOG_DIR isn't getting globally updated
+    _log_dir="${1:-"${LOG_DIR:="$(mktemp -d)"}"}"
 
-        inotifywait -e open "$_watch_file"
+    [ -d "$_log_dir" ] || mkdir -p "$_log_dir"
 
-        read -r _mount_point < "$_watch_file"
-
-        [ -n "$_mount_point" ] || continue
-    done
-
-    log "Mountpoint is ${_mount_point}."
-    find_assertions "${_mount_point}"
+    _log_path="${_log_dir}/$(date -Iseconds).log"
```

Comment:
`_log_path` feels like a misnomer as `_log_dir` should already be the path. Should we use `_log_file` instead, with it being just the name and have `_log_path` be `${_log_dir}/${_log_file}`?